tests/conftest.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+
+import pytest
+from brownie import CurveTokenV1, CurveTokenV2, CurveTokenV3
+from brownie.test import given, strategy
+from hypothesis import settings
+
+pytest_plugins = [
+    "fixtures.accounts",
```

Comment:
Separating fixtures across multiple modules is useful for large test suites, but for a project of this size I feel it's overkill.  I would keep all fixtures within the base `conftest.py`